lib/cover_my_meds/railtie.rb

```diff
@@ -24,9 +24,11 @@ def configured_client(api_id, secret = nil)
 
     private
     def credentials
-      api_id = Rails.application.secrets.cmm_api_id || ENV['CMM_API_ID']
-      secret = Rails.application.secrets.cmm_api_secret || ENV['CMM_API_SECRET']
-      [ api_id, secret ]
+      if Rails.application.respond_to? :secrets
+        api_id = Rails.application.secrets.cmm_api_id
+        secret = Rails.application.secrets.cmm_api_secret
+      end
+      [ api_id || ENV['CMM_API_ID'], secret || ENV['CMM_API_SECRET'] ]
```

Comment:
Maybe it's just me, but I find this syntax messy (too much happening inside brackets).  Could you put it in an else?  Or find someone who is willing to dispute my claim, that works too. :)